### ISSUE TITLE
AutoRotation not working after 6.5 ?

### ISSUE THREAD

**Issue by Romaain002** [awgil/ffxiv_bossmod#184]
Hi, the AutoRotation is not working, I imagine you're working on an update? :D

**Comment by weapi1** [comment]
Works fine for me

**Comment by haganebr** [comment]
Yep. I had the same problem, but this morning's update fixed it. Issue can probably be closed.
